next.config.js

```diff
@@ -19,6 +19,10 @@ const moduleExports =  {
     ]
   },
 
+  env: {
+    releaseVersion: 'full', // feature flag - mvp or full
```

Comment:
As a followup of adding the feature flag, a ticket would need to be created to remove the flag post-full release, which would include
 - clean up usages of the feature flag
 - remove the feature flag (definition and documentation)

I'll create this ticket and put it at the bottom of the product backlog.

Reply:
Thanks @Bhavesh-Patel.

Reply:
@Bhavesh-Patel has created the [ticket](https://trello.com/c/EqV5prNL/246-remove-mvp-feature-flag-after-full-service-release):